sandbox/removeRows.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+
+data = pd.read_csv(
+    "/usr/local/google/home/bumaschny/synthea/src/main/resources/providers/hospitals.csv"
```

Comment:
Please use relative file path. 
for example: 
`../synthea/src/main/resources/providers/hospitals.csv`